CHANGELOG.md

```diff
@@ -1,9 +1,19 @@
 # Changelog
 
+## [0.2.0] ｜ 2025-08-23
+
+### Features
+- IntelliSense: environment variable suggestions (names only) for:
+  - JS/TS: `process.env.FOO`, `process.env['FOO']`, `process.env["FOO"]`
+  - Python: `os.environ['FOO']`, `os.environ.get('FOO')`, `os.getenv('FOO')`
+- Hover: when hovering over the name, displays `VARIABLE=VALUE`
+- Variable discovery from `process.env` and `.env` files in the workspace (with a watcher to update automatically)
+
 ## [0.1.0] ｜ 2020-10-13
 
 ### Features
 
 ↳ [`53e6022`](https://github.com/IronGeek/vscode-env/commit/53e6022cb0fbea99d3e6a678cd610904c18f548d) initial commit
 
+[0.2.0]: https://github.com/HellFiveOsborn/vscode-env/releases/tag/v0.2.0 "v0.2.0"
```

Comment:
Should be: `https://github.com/IronGeek/vscode-env/releases/tag/v0.2.0 "v0.2.0"`